web_widget_json_graph/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Web Widget JSON Graph",
+    "version": "12.0.1.0.0",
+    "author": "Vauxoo, Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "category": "Hidden/Dependency",
+    "website": "https://github.com/OCA/web/tree/12.0/web_widget_json_graph",
```

Comment:
```suggestion
    "website": "https://github.com/OCA/web",
```

Reply:
@marcelsavegnago done. However, AFAIK they're equivalent, at least that's [what the template says](https://github.com/OCA/maintainer-tools/blob/0290f3971ed057ff868731ddac36a28cf4b1bc97/template/module/__manifest__.py#L10)